assignment-6/src/Components/SearchBar.js

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+
+const SearchBar = ({ userData, setUserData }) => {
+
+    function searchUser(searchText) {
+        // console.log(searchText);
```

Comment:
When the search text does not match any of the users in the user list, no results are returned. 
It would be great if you have time you can implement that scenario as well.